### PR TITLE
fix: lint errors

### PR DESCRIPTION
**Pull request by dkoshkin**
Fixing errors introduced by https://github.com/mesosphere/konvoy-image-builder/pull/57

### CODE REVIEW

File: cmd/konvoy-image-wrapper/cmd/wrapper.go [hunk 1]
```diff
@@ -307,7 +307,7 @@ func (r *Runner) setUserMapping() error {
 		r.homeDir,
 	)
 	//nolint:gosec // file must be world readable
-	return ioutil.WriteFile(filePath, []byte(content), 0644)
+	return ioutil.WriteFile(filePath, []byte(content), 0o644)
```
Comment (alejandroEsc) [inline cmd/konvoy-image-wrapper/cmd/wrapper.go:310]: interesting

File: docs/cli/konvoy-image_provision.md [hunk 2]
```diff
@@ -9,14 +9,17 @@ konvoy-image provision <inventory.yaml|hostname,> [flags]
 ### Examples
 
 ```
-build ./inventory.yaml
+provision --inventory-file inventory.yaml images/generic/centos-7.yaml
```
Comment (alejandroEsc) [inline docs/cli/konvoy-image_provision.md:12]: nice, thank you

**Review by alejandroEsc — APPROVED** [review]
thank you, but how did we get through main with these liint issues?

**Comment by dkoshkin** [comment]
> thank you, but how did we get through main with these liint issues?

Looks like it was merged with errors :/

**Pull request merged**